module/mailnews/newsread.c

```diff
@@ -530,3 +536,12 @@ void ListView_SetHeaderSortImage(HWND hListView, int sortedColumn, BOOL sortAsce
       Header_SetItem(hHeader, i, &hdi);
    }
 }
+
+/*
+* Resets the news list to it's default sort: date descending
+*/
+void ResetListSort(HWND hListView)
+{
+   ListView_SortItems(hListView, CompareListItems, -3); // 1-based column
```

Comment:
These magic numbers are hard to understand.  Can you use the enum constants instead?

Reply:
🤦🏻‍♂️